samples/matter/manufacturer_specific/README.rst

```diff
@@ -0,0 +1,275 @@
+      You can press a **Button 1** to start the factory reset when needed.
+      **Button 2** and **Button 3** are used to toggle the state of ``NordicDevkit`` cluster's attributes, ``LED2`` and ``LED3`` respectively.
+      Matter command ``SetLED`` is used to control the state of  ``LED2`` and ``LED3``.
+      It takes two arguments: the LED number (``2`` or ``3``for ``LED2`` or ``LED3`` attribute) and the action to be performed (``0`` to turn the LED off, ``1`` to turn it on, ``2`` to toggle the state).
+      **LED 2** and **LED 3** reflect the state of the ``LED2`` and ``LED3``.
+      ``NordicDevkit`` cluster Additionally introduces a writable ``DevKitName`` attribute, of string type.
+
+   .. group-tab:: nRF54 DKs
+
+      The sample starts the Bluetooth® LE advertising automatically and prepares the Matter device for commissioning into a Matter-enabled Thread network.
+      The sample uses an **LED 0** to show the state of the connection.
+      You can press a **Button 0** to start the factory reset when needed.
+      **Button 1** and **Button 2** are used to toggle the state of ``NordicDevkit`` cluster's attributes, ``LED2`` and ``LED3`` respectively.
+      Matter command ``SetLED`` is used to control the state of  ``LED2`` and ``LED3``.
+      It takes two arguments: the LED number (``2`` or ``3`` for ``LED2`` or ``LED3`` attribute) and the action to be performed (``0`` to turn the LED off, ``1`` to turn it on, ``2`` to toggle the state).
+      **LED 1** and **LED 2** reflect the state of the ``LED2`` and ``LED3``.
+      ``NordicDevkit`` cluster Additionally introduces a writable ``DevKitName`` attribute, of string type.
+
+
+Custom manufacturer-specific cluster
+====================================
+The sample provides a custom ``NordicDevkit`` cluster, configured in :file:`src/default_zap/NordicDevKitCluster.xml` file.
+To learn more about adding custom clusters to your Matter application, see the :ref:`ug_matter_creating_accessory_vendor_cluster` section.
+
+.. _matter_manufacturer_specific_network_mode:
+
+Remote testing in a network
+===========================
+
+.. |Bluetoothsc| replace:: Bluetooth®
+
+By default, the Matter accessory device has no IPv6 network configured.
+You must pair it with the Matter controller over |Bluetoothsc| LE to get the configuration from the controller to use the device within a Thread network.
+The controller must get the `Onboarding information`_ from the Matter accessory device and provision the device into the network.
+For details, see the `Commissioning the device`_ section.
+
+Configuration
+*************
+
+|config|
+
+.. _matter_manufacturer_specific_custom_configs:
+
+Matter manufacturer specific sample with Trusted Firmware-M
+===========================================================
+
+.. include:: ../template/README.rst
+    :start-after: matter_template_build_with_tfm_start
+    :end-before: matter_template_build_with_tfm_end
+
+Device Firmware Upgrade support
```

Comment:
Please check my https://github.com/nrfconnect/sdk-nrf/pull/20647. I hope it goes in first. Then you can copy from that how the "Advanced configuration options" chapter is added.

Reply:
thanks, aligned section titles, but can't include default `/includes/advanced_conf_matter.txt` there as this sample does not add `FEM support`